.storybook/preview.js

```diff
@@ -0,0 +1,24 @@
+import * as NextImage from 'next/image'
+import { WalletProvider } from '../components/WalletProvider'
+import { theme } from '../theme'
+
+// de-optimize Image component
+const OriginalNextImage = NextImage.default
+Object.defineProperty(NextImage, 'default', {
+  configurable: true,
+  value: (props) => <OriginalNextImage {...props} unoptimized />,
+})
+
+export const parameters = {
+  chakra: {
+    theme,
+  },
```

Comment:
`@chakra-ui/storybook-addon` reads options from `parameters.chakra`. We pass our `theme` here so our stories match our theme.